Layout.py

```diff
@@ -421,63 +483,64 @@ def do_run(self):
         })
 
 
-class DestroyCurrentPaneCommand(RevocableCommand):
+class LayoutDestroyCurrentCommand(RevocableCommand):
     def do_run(self):
         self.destroy_pane(self.current_group)
 
 
-class CloneFileToPaneCommand(RevocableCommand):
+class LayoutCloneCommand(RevocableCommand):
     def do_run(self, direction):
         self.clone_file_to_pane(direction)
 
 
-class CarryFileToPaneCommand(RevocableCommand):
+class LayoutCarryCommand(RevocableCommand):
     def do_run(self, direction):
         self.carry_file_to_pane(direction)
 
 
-class ResizePaneCommand(PaneCommand):
-    def run(self, direction, step=1):
-        self.resize_pane(self.current_group, direction, step / 100)
+class LayoutResizeCommand(PaneCommand):
+    def run(self, direction, step=1, alt=False):
+        self.resize_pane(self.current_group, direction, step / 100, alt)
 
 
 class LoadLayoutFromCommand(PaneCommand):
     def load_layout_from_file(self, filename):
         layout = open(self.get_layout_path(filename), 'r').read()
         self.load_layout_from_json(layout)
+        sublime.status_message('Load Layout named `%s`' % filename)
+        self.refresh_layout()
 
     def run(self, filename=None):
         if filename:
             self.load_layout_from_file(filename)
         else:
-            view = self.window.show_input_panel(
+            self.view = sublime.active_window().show_input_panel(
                 caption='Load from Layout File',
                 initial_text='',
                 on_done=self.load_layout_from_file,
                 on_change=None,
                 on_cancel=None
             )
-        sublime.status_message('Load Layout named `%s`' % filename)
```

Comment:
I didn't remove it, I moved it above, in the 'load_layout_from_file', to fix the 'no focus to panel'